tests/broad/scala_test/src/main/scala/org/broadinstitute/dsp/pipelines/commandline/ConfigParser.scala

```diff
@@ -910,4 +910,83 @@ class ConfigParser
           )
         }
     )
+
+  note("")
+  cmd(RNAWithUmis.entryName)
+    .text("Test the RNA with UMIs workflow")
+    .action(
+      (_, config) =>
+        config.copy(
+          test = RNAWithUmis
+      )
+    )
+    .children(
+      opt[WorkflowTestCategory]('t', "test")
+        .text("The type of test to run")
+        .optional()
+        .action { (test, config) =>
+          config.copy(
+            rnaWithUmisConfig = config.rnaWithUmisConfig.copy(category = test)
+          )
+        },
+      opt[String]('b', "branch")
+        .text("The branch of truth data to test against (Defaults to master)")
+        .optional()
+        .action { (branch, config) =>
+          config.copy(
+            rnaWithUmisConfig =
+              config.rnaWithUmisConfig.copy(truthBranch = branch)
+          )
+        },
+      opt[CromwellEnvironment]('e', "env")
+        .text(
+          s"The environment that this should run in ${CromwellEnvironment.optionsString}"
+        )
+        .required()
+        .action { (env, config) =>
+          config.copy(
+            rnaWithUmisConfig = config.rnaWithUmisConfig.copy(env = env)
+          )
+        },
+      opt[Unit]("update-truth")
+        .text(
+          "Update the truth data with the results of this run."
+        )
+        .optional()
+        .action { (_, config) =>
+          config.copy(
+            rnaWithUmisConfig =
+              config.rnaWithUmisConfig.copy(updateTruth = true)
+          )
+        },
+      opt[String]("use-timestamp")
+        .text(
+          "Do not run the workflows. Instead, just use a previous runs timestamp (yyyy-MM-dd-HH-mm-ss)"
+        )
+        .optional()
+        .action { (timestamp, config) =>
+          config.copy(
+            rnaWithUmisConfig = config.rnaWithUmisConfig
+              .copy(useTimestamp = Option(timestamp))
+          )
+        },
+      opt[Unit]('u', "uncached")
+        .text("Disable call-caching for the main workflow run")
+        .optional()
+        .action { (_, config) =>
+          config.copy(
+            rnaWithUmisConfig =
+              config.rnaWithUmisConfig.copy(useCallCaching = false)
+          )
+        },
+      opt[PapiVersion]("papi-version")
```

Comment:
We probably don't need PAPI version anymore since we cannot run on PAPIv1 any longer

Reply:
Oops - too much copying and pasting!